docs/guides/references/configuration.mdx

```diff
@@ -215,6 +215,7 @@ object:
 | `excludeSpecPattern`           | `['/snapshots/*', '/image_snapshots/*']`  | A String or Array of glob patterns used to ignore spec files that would otherwise be shown in your list of specs. [Please read the notes on using this.](#excludeSpecPattern)                                                                  |
 | `experimentalSingleTabRunMode` | `false`                                   | Run all specs in a single tab, instead of creating a new tab per spec. This can improve run mode performance, but can impact spec isolation and reliability on large test suites. This experiment currently only applies to Component Testing. |
 | `slowTestThreshold`            | `250`                                     | Time, in milliseconds, to consider a test "slow" during `cypress run`. A slow test will display in orange text in the default reporter.                                                                                                        |
+| `devServerPublicPathRoute`     | `/__cypress/src`                          | The public path of the dev server in use. We recommend using the Cypress default, but may need to be changed if needed. See [Framework Configuration](/guides/component-testing/component-framework-configuration) or dev server specific documentation (in particular `@cypress/vite-dev-server`).                                                                |
```

Comment:
Suggest

... but may need to be changed if needed. =>

... but may need to be changed.

Sounds otherwise a bit awkward with 2 x need.

Also it looks like prettier doesn't like the misaligned table formatting.

Reply:
It's definitely a bit awkward since there isn't a great place to currently reference the configuration option. It's mostly supposed to be used internal by the bundler dev servers, but we also expose it so people can customize a dev server. Got this updated in [1f5f96c](https://github.com/cypress-io/cypress-documentation/pull/5830/commits/1f5f96c4dbe5e1bb5522dac9804c14db3cf30e03) if you want to take another look!